media/utils/setLoading.js

```diff
@@ -16,12 +16,6 @@ function setError() {
     </div>
     `);
   addActionButtons();
-  $("#ghHolder").append(
```

Comment:
So new users won't see it either?
How do they discover usage patterns?

Reply:
Actually, I was also thinking about this. The extension walktrhough is an answer, isn't it?

Reply:
I personally have never used that walk through, or read the changelog that pops up.

Reply:
I'd prefer to leave it in the Initial HTML, and when reloading, remove it

Reply:
Gotcha, good idea. Let me do the changes.